setup.py

```diff
@@ -27,6 +27,7 @@
     "pytest>=2.8.0",
     'pytest-mock>=2.0,<3.0 ; python_version=="2.7"',
     'pytest-mock>=3.0 ; python_version>="3"',
+    "tox>=3.21.0",
```

Comment:
Actually we cannot leave it like this, since tox -e py27 would like to install tox, as part of the run, and it seems to fail for me for py27... So let's leave it out for simplicity.  Otherwise we would have to do the `python_version` dance, but it is probably not worth it.